### ISSUE TITLE
fuge 2.0.0 cant't set environment process env

### ISSUE THREAD

**Issue by Remexllee** [apparatus/fuge#161]
I found out an issue when I update fuge to 2.0.0, my `environment` parameter can't be set into process env.

The yml file below work well in fuge 1.0.0
But in 2.0.0, there is no env for `BOOTSTRAP_ENV` and `BOOTSTRAP_DC`

```
fuge_global:
  ...
  dns_host: 127.0.0.1
  monitor_excludes: 
    - '**/node_modules/**'
  environment:
   - "BOOTSTRAP_ENV=abc"
   - "BOOTSTRAP_DC=def"
   ...
```

**Comment by pelger** [comment]
Ah yes - have a fix for that will publish shortly!

**Comment by pelger** [comment]
Fixed and published v 2.0.1 can you please reinstall and test - thanks!

**Comment by Remexllee** [comment]
New version(2.0.1) works great in my local, thanks for fixed.
